### PR TITLE
Coverage

### PR DESCRIPTION
**Pull request by robertsLando**

### CODE REVIEW

File: test/will.js [hunk 1]
```diff
@@ -113,6 +113,24 @@ test('delivers old will in case of a crash', function (t) {
...
+  })
+  t.tearDown(broker.close.bind(broker))
+
+  var brokerId = 'dummyBroker'
+
+  broker.brokers[brokerId] = Date.now() - heartbeatInterval * 3.5
+
+  setTimeout(() => {
+    t.equal(broker.brokers[brokerId], undefined, 'Broker deleted')
+  }, heartbeatInterval * 4)
```
Comment (mcollina) [inline test/will.js]: can we use http://npm.im/lolex to test this?

Comment (robertsLando) [reply test/will.js]: Why? BTW lolex is deprecated in favor of https://github.com/sinonjs/fake-timers

Comment (mcollina) [reply test/will.js]: timers are highly unreliable in CI.

Comment (robertsLando) [reply test/will.js]: I don't really understand how they work. I have tried to simply add 

```js
require("@sinonjs/fake-timers").install()
```

To `test/helper.js` but seems timers are not working. I have also tried with other methods but I can't make them work

**Comment by robertsLando** [comment]
@mcollina Are you ok if we land this and we do another one for timers fix?

**Comment by mcollina** [comment]
I would prefer to fix the timers.

**Comment by robertsLando** [comment]
@mcollina Is it ok?

**Pull request merged**